app/uk/gov/hmrc/cataloguefrontend/connector/ServiceDependenciesConnector.scala

```diff
@@ -87,8 +87,11 @@ class ServiceDependenciesConnector @Inject() (
       "artefact"     -> artefact,
       "versionRange" -> versionRange.range
     )
+
+    val scopeParams = scopes.map( scope => s"scope=${scope.asString}" ).mkString("&")
```

Comment:
you can add the scopes to the queryParams map above `"scope" -> scopes.map(_.asString)` and the [uri interpolator](https://sttp.softwaremill.com/en/latest/model/uri.html) will handle them